package.json

```diff
@@ -58,12 +60,12 @@
     "eslint-plugin-react": "^7.21.3",
     "eslint-plugin-react-hooks": "^4.1.2",
     "jest": "^26.5.0",
-    "postcss": "^7.0.32",
+    "postcss": "^7.0.35",
```

Comment:
Doh 🤦 Okay, this is totally my fault. [In our thread](https://fishtownanalytics.slack.com/archives/C01BLFXN8QN/p1613673961009400), I meant to say, _let's do the Storybook workaround so we can use PostCSS 8_, not use the compatibility work around. I even thumbs up'd the compatibility decision! So sorry!

If it's possible for us go with PostCSS 8, I think that's what we should do! Tailwind is the meat & potatoes of FUI, Storybook is a tool to support the project. So, I think favoring Tailwind's dependencies in this case is preferable.

If it's not easy to do the Storybook workaround for now, we can leave this as is.

Reply:
@eiof and I talked about this offline. It looks like Storybook's alpha and beta versions aren't super stable. I had no luck getting them to work on their own without the tailwind changes. We determined that it's ok for now to go forward with the tailwind 2 compatibility build. The [compatibility build](https://tailwindcss.com/docs/installation#post-css-7-compatibility-build) gives us the same features as the regular build! Updating this to the regular when all of our dependencies can handle it should be super simple.